### ISSUE TITLE
Create New ROW Performance Measure Report (TL)

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#1872]
### ROW Performance Measure Custom Report

**Stakeholder:** Paloma, Kim
**DTS Manager:** Tracy

Description: Report Parameters:
- Number of Permits Submitted (DS, EX, RW (Minus the Street Event subtype))
- Number of Permits Issued

Request ID: DTS18-100405

**Comment by atdservicebot** [comment]
*From atdservicebot on 2019-05-10*:

Ryan called for an update on this request. Urgency level has been updated and item moved to the top of my task list. 

Created By: Tracy Linder
Created Date: 2018-12-05 13:00:00

**Comment by TracyLinder** [comment]
Tracy has some partial queries that will help with this. Email her when you're read for them.

**Comment by TracyLinder** [comment]
Email to Sree:
Hi Sree,

I talked to Paloma about this; she was in my next meeting so I didn’t need an email.  Notes below, and queries attached. 

•	Paloma would prefer to query by date and view results grouped by folder type ,and would to view RW subtypes grouped also. 
•	They are no longer report on SCP, so we can take all SCP out of this request. 

Thank You,

Tracy Linder, Product Manager

**Comment by TracyLinder** [comment]
Sree: Sent Tracy a template this morning for review.

**Comment by TracyLinder** [comment]
**Sree sent us an example draft report for review on 6/8**
![image.png](https://images.zenhubusercontent.com/5cca1e522b8dfc0ee14dedf5/5e12965b-f2db-4fef-87ca-a6691b5d691d)





**My email response to Sree:**

Hi Sree,

The report looks nice. I do request one change: please remove the subcode column.

Thank You, 

Tracy Linder, Product Manager

**Comment by TracyLinder** [comment]
Sree will be moving this to Test, so we can test the data is populating correctly.

**Comment by TracyLinder** [comment]
Sree sent the deployment plan for this to @Nadin-Nader on 6/11.
Nate, should I create an issue for the deployment?

**Comment by Nadin-Nader** [comment]
@TracyLinder Sree and i will try to deploy to DEV (Currently Trying). We got it on DEV now we just need to get it to pull the data. Once that is done I can create the deployment to CTM Tango

**Comment by Nadin-Nader** [comment]
@TracyLinder @Sreelatha-BlazeKannan got the report to work and deployed to DEV.
Task for the TEST deployment has been created and sent to AMANDA_SUPP (SCTASK0075805)

**Status:**

Waiting on deployment to TEST and UAT on TEST. Then if all is well, deployment to PROD.

**Comment by Nadin-Nader** [comment]
**Status**
Currently with the AMANDA CTM Tango team SCTASK0075805. Just waiting on deployment

**Comment by Sreelatha-BlazeKannan** [comment]
Nadin Renamed 56210 report code and sent to Surya for  TEST deployment

**Comment by Nadin-Nader** [comment]
Updated the Parameters for Surya on DEV and sent her an email asking her to proceed with the deployment.

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan Sent email to Surya:

> Surya,
> 
> I looked in the Reports>ATD menu and didn’t find it in TEST. I went ahead and added it to the ATD Menu on DEV and TEST and added the users as well. I was then able to run the report successfully.
> Thank you for your help. I appreciate it.
> 

![image.png](https://images.zenhubusercontent.com/5e8dfbfabf171441c3a5684e/2d0fbf32-0562-4cc9-a827-525623c7f987)

Next Step PROD?

**Comment by TracyLinder** [comment]
awesome @Nadin-Nader! I'll test this ASAP, and then hopefully move to PROD

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan @TracyLinder;  I was able to run this on TEST. I created the task for CTM and Surya to deploy to PROD SCTASK0078266

**Comment by TracyLinder** [comment]
@Nadin-Nader this wasn't ready for PROD when you submitted the deployment ticket. It's ready now though; i just finished testing. I'll make this an epic & create an issue for DevOps.

**Comment by Sreelatha-BlazeKannan** [comment]
As per Nadin this will be planned to move from TEST to PROD on 07/07/2020.

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan @TracyLinder  This was deployed to PROD by Gita on 7/7/2020

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan  @TracyLinder  This is on PROD now and i was able to run the report successfully. Close card when Ready

**Comment by TracyLinder** [comment]
@Sreelatha-BlazeKannan @Nadin-Nader 

We just ran this in Prod and the numbers (totals) are not correct. This is because the query parameters are querying the wrong data. See image:

![image](https://user-images.githubusercontent.com/49930868/87190412-090f7380-c2b8-11ea-9acf-fea5803f1580.png)

**Comment by TracyLinder** [comment]
We're going to close this one & create a new ticket for the parameter fix.
